.github/workflows/performance.yml

```diff
@@ -33,7 +33,7 @@ jobs:
   # Runs the performance test suite.
   performance:
     name: Performance tests ${{ matrix.memcached && '(with memcached)' || '' }}
-    uses: WordPress/wordpress-develop/.github/workflows/reusable-performance.yml@trunk
+    uses: ./.github/workflows/reusable-performance.yml
```

Comment:
Note to reviewers: This is temporary, so that the PR uses the workflow file from this branch (as it is being modified) instead of that from `trunk`.